repos/system_upgrade/el7toel8/actors/pcidevicesscanner/tests/test_pcidevicesscanner.py

```diff
@@ -1,11 +1,9 @@
-import pytest
-
 from leapp.models import PCIDevices, PCIDevice
 from leapp.libraries.actor.pcidevicesscanner import parse_pci_devices, produce_pci_devices
 
 
-def test_parse_pci_devices(current_actor_libraries):
```

Comment:
Removed everywhere this fixture, because it is not needed, the actor context automatically contains relevant libs